build_runner/lib/src/entrypoint/options.dart

```diff
@@ -539,6 +548,67 @@ class _TestCommand extends BuildRunnerCommand {
   }
 }
 
+class _CleanCommand extends Command<int> {
```

Comment:
at some point we need to consider splitting up this file... Doesn't need to happen now though.

Reply:
Ya, we would probably want to use part files though to keep these commands private (at least for now?), and I would personally rather deal with a large file than part files most of the time :man_shrugging: .